### ISSUE TITLE
FSI fails to load generated load script ifle 

### ISSUE THREAD

**Issue by chinwobble** [fsprojects/Paket#3353]
### Description

FSI fails to load generated load script file

### Repro steps

Run the following commands with forge on W10 and paket 5.177.1
1. `forge paket init`

2. `forge paket add FSharp.Control.FusionTasks`

3. `forge paket generate-load-scripts`

4. run the following fsx file 
```
#load @".paket\load\netcoreapp2.1\main.group.fsx"
```

### Expected behavior

Should be able to load main package script 

### Actual behavior

Get the following output when trying to load the script 
```
main.group.fsx(27,1): error FS0074: The type referenced through 'System.Threading.Tasks.ValueTask' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Threading.Tasks.Extensions'.
```

### Known workarounds
none

**Comment by smoothdeveloper** [comment]
Thanks for the report @chinwobble, this may be related to https://github.com/Microsoft/visualfsharp/issues/3309

If you can, would you try loading this way:

```fsharp
#load @".paket\load\net462\main.group.fsx"
```

FSI is only supported on full .net framework AFAIK, if you need to reference netstandard binaries, you'll need to use work around detailled in the visualfsharp issue I mentioned above.

**Comment by matthid** [comment]
Yes @smoothdeveloper is correct but just to confirm loading a netcoreapp load script into full framework fsi is never a safe operation, even when visualfsharp fixes the current problems. As indicated you need to generate and use the full framework load script
